pennylane/interfaces/jax_jit.py

```diff
@@ -17,6 +17,7 @@
 """
 
 # pylint: disable=too-many-arguments
+from doctest import NORMALIZE_WHITESPACE

```

Comment:
How come this was added here?

Reply:
Thank you for the review @antalszava. This is added automatically by vscode. removed!